pkgs/by-name/cn/cnsprcy/package.nix

```
@@ -5,19 +5,29 @@
   sqlite,
 }:
 
-rustPlatform.buildRustPackage {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "cnsprcy";
-  version = "0.2.0";
+  version = "0.3.0";
```

We also need to rename the commit and PR title and regenerate the hashes

```suggestion
  version = "0.3.1";
```